src/Field.jsx

```diff
@@ -0,0 +1,77 @@
+import {
+    Grid,
+    InputAdornment,
+    makeStyles,
+    MenuItem,
+    Select,
+    Switch,
+    TextField,
+    Typography
+} from '@material-ui/core';
+import ReactDOM from "react-dom";
+import React from "react";
+
+
+export default function Field (props) {
+    function handleChange(event) {
+        props.updateParent(props.metadata.name, event.target.value, !props.isValueValid(event.target.value));
+    }
+    function handleSwitch(event) {
+        props.updateParent(props.metadata.name, event.target.checked ? 1 : 0, false);
+    }
```

Comment:
This doesn't seem to be used anywhere

Reply:
Thanks for pointing this out, it's leftover code. I removed it together with the import of the Switch component.